src/emulator/net/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+add_library(
+net
+STATIC
+include/net/functions.h
+include/net/state.h
+src/net.cpp
+)
+
+target_include_directories(net PUBLIC include)
+target_link_libraries(net PUBLIC sdl2 vita-headers)
+if (WIN32)
+    target_link_libraries(net PRIVATE winsock)
```

Comment:
Oh, this should be `PUBLIC` because net/state.h includes winsock headers. I wonder would that fix the define issue.

Then again, if we can get rid of the headers out of the net/state.h header, we could keep this PRIVATE.